15-symfony-flex-platformsh.yaml

```diff
@@ -1,85 +0,0 @@
-requirements:
-  - type: file_exists
-    value: composer.json
-  - type: has_composer_package
-    value: symfony/flex
-  - type: has_composer_package
-    value: symfonycorp/platformsh-meta
```

Comment:
this was the only significant diff with 30-symfony-flex.yaml, yet we don't use this meta anymore (there is no recipe attached to it)